addOns/quickstart/CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Added
 - Stats counter to the main toolbar button (Issue 8375).
 
+### Fixed
+- An NPE that might occur when switching look and feel.
```

Comment:
I'd say something like "An exception that prevented the look and feel from changing completely." If not at least change NPE to exception and replace "might occur" with "occurred" (it would always).